index.php

```diff
@@ -53,12 +53,13 @@
 
 	<ul id="sidebar" class="animated slideInLeft">
 		<li><i class="fa fa-2x fa-fw allsky-constellation" id="overlayBtn" title="Show constellations overlay" ng-click="toggleOverlay()" ng-class="{'active': showOverlay}"></i></li>
-		<li><a href="videos" title="Archived Timelapses"><i class="fa fa-2x fa-fw fa-play-circle"></i></a></li>
-		<li><a href="keograms" title="Archived Keograms"><i class="fa fa-2x fa-fw fa-barcode"></i></a></li>
-		<li><a href="startrails" title="Archived Startrails"><i class="fa fa-2x fa-fw fa-circle-notch"></i></a></li>
+		<li><a href="show_thumbnails.php?dir=videos&prefix=allsky&title=Timelapse%20Videos" title="Archived Timelapses"><i class="fa fa-2x fa-fw fa-play-circle"></i></a></li>
```

Comment:
I do like the old url better. It is easier to remember `/videos`, `/startrails` and `/keograms` are easier to remember and nicer when sharing the URL.

I understand you've refactored to remove duplicated code but it there a way we keep the urls intact and call the unified code inside the respective index.php?

Reply:
I made the changes - added 3 */index.php files and modified index.php, functions.php, and show_thumbnails.php.